x/wasm/internal/types/msg.proto

```diff
@@ -10,49 +10,75 @@ option go_package = "github.com/CosmWasm/wasmd/x/wasmd/internal/types";
 option (gogoproto.goproto_getters_all) = false;
 
 
+// MsgStoreCode submit WASM code to the system
```

Comment:
```suggestion
// MsgStoreCode submit Wasm code to the system
```

Since Wasm is not an acronym, it is spelled like this (see https://webassembly.org/)